catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -187,6 +194,11 @@
     public static final String DESCRIPTION = "description";
 
     /**
+     * The default tag type for all metacards
+     */
+    public static final String DEFAULT_TAG = "catalog";
```

Comment:
I wonder if the default should be `resource` based on the JavaDoc for `Metacard`.